test/new/db/cmd/cmd_repeats

```diff
@@ -3,14 +3,10 @@ FILE=-
 EXPECT=<<EOF
 16
 18
-16
-18
 EOF
 CMDS=<<EOF
 @@?~?
 @@@?~?
-?@@?~?
```

Comment:
these additional help commands are not necessary. Help commands follow a very strict rule which is `<command-name>?`, so having `?@@?` is not consistent.

Reply:
agree, but it was confusing for some people was expecting to get the help of the modifiers as arguments of the ? command. but i agree